src/app/(proper_react)/(redesign)/(public)/breach-details/[breachName]/BreachDetailView.tsx

```diff
@@ -114,7 +114,7 @@ export const BreachDetailsView = (props: Props) => {
         {typeof breach.Domain === "string" && breach.Domain.length > 0 ? (
           <p>
             <TelemetryLink
-              href={breach.Domain}
+              href={`//${breach.Domain}`}
```

Comment:
Ah yes... the HIBP `Domain` does not have a protocol. We could probably just hardcode this to `https://`protocol anyways since they now consider [protol-relative URLs an anti-pattern](https://www.paulirish.com/2010/the-protocol-relative-url/).

```suggestion
              href={`https://${breach.Domain}`}
```

```sh
curl -s https://haveibeenpwned.com/api/v3/breaches | npx prettier --parser=json | grep "Domain" | sort -i | uniq -c
  38     "Domain": "",
   1     "Domain": "000webhost.com",
   1     "Domain": "123rf.com",
   1     "Domain": "126.com",
   1     "Domain": "163.com",
   1     "Domain": "17173.com",
   1     "Domain": "17app.co",
...
```

I haven't looked at this in a while, but I think it was still 30-40% errors whether we defaulted to http:// or https:// protocol, and there are still just a number of these old breached sites that are no longer resolving/registered and 404 for other reasons. I still wonder how much benefit there is to us or users to link to breached sites. On one hand, a lot of the sites are "meh" and shady. Definitely don't think there is enough value in registering any of these expired Domains and redirecting them to monitor.mozilla.org. On the other hand, a convenience link to adobe.com to change your URL could be useful, so I'm firmly in the 🤷 camp.

Reply:
Correction, was 27% broken outbound links, circa 2019, per this wise oracle: https://github.com/mozilla/blurts-server/issues/1362#issuecomment-556875424

Reply:
Thank you for citing the article! Good to know!